src/lib/turtle.js

```diff
@@ -1209,7 +1234,7 @@ function generateTurtleModule(_target) {
             return this._managers[type];
         };
 
-        proto.reset = function() {
+        proto.reset = function(keepShowingUI) {
```

Comment:
Maybe document the new flag somewhere?  I'm guessing it's optional? (and default is false when value is missing).

Reply:
Yes you're right. I'll do so.